setup.py

```diff
@@ -12,11 +12,11 @@
     description="Graph analysis for neuropycon (using nipype, and ephypype); based on previous packages dmgraphanalysis and then dmgraphanalysis_nodes and graphpype",
     lisence='BSD 3',
     install_requires=['numpy>=1.3.0',
-                      'statsmodels',
+                      'statsmodels==0.8.0',
                       'patsy',
                       'nipype',
                       'configparser',
                       "pandas",
                       'networkx==1.9',
-                      "matplotlib"]
+                      "matplotlib==2.2.3"]
```

Comment:
you should not force versions like this. Then users will have to decide between using `neuropycon` and a new version of `matplotlib` ...

Reply:
this is exactly what I tried to avoid, but have seen the link I put earlier? There seems to be an issue with the Windows version of matplotlib.
Do you have another solution?

Reply:
hmm ... but didn't we stop supporting Windows? From what I understood `nipype` does not really work on Windows ...

Reply:
ping @dmalt -- he may have an opinion here

Reply:
Well so far we haven't done any modification in graphpype.